doc/source/admin/galaxy_options.rst

```diff
@@ -2636,6 +2636,19 @@
 :Type: float
 
 
+~~~~~~~~~~~~~~~~~
```

Comment:
This got included with the config rebuild.

Reply:
if you're going to drop the symlink can you also unstage this ? This is missing in 20.09, I can take care of that (making a list of papercuts to backport now).